__tests__/run_seed.test.js

```diff
@@ -245,3 +253,31 @@ describe("POST", () => {
     });
   });
 });
+describe("GET : users", () => {
+  test('Returns a 200 status and an array of all users', () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body}) => {
+      expect(body.users.length).toBeGreaterThan(0);
+      body.users.forEach((user) => {
+        expect(user).toEqual(
+          expect.objectContaining({
+            username : expect.any(String),
+            name : expect.any(String),
+            avatar_url : expect.any(String)
+          })
+        )
+      })
+    })
+  })
+  test("Responds with a 404 error message when passed an incorrect route", () => {
```

Comment:
You don't need this in addition to the test up on line 33 - they're testing for the same thing. An incorrect route  is the same regardless of the endpoint that you're currently working on.